pmoves/services/hi-rag-gateway-v2/clients/neo4j.py

```
@@ -0,0 +1,90 @@
+            logger.exception("warm loop error")
+        time.sleep(max(15, NEO4J_DICT_REFRESH_SEC))
+
+
+import threading as _t
+if driver is not None:
+    _t.Thread(target=warm_loop, daemon=True).start()
+
+
+def graph_terms(query: str, limit: int = 8, entity_types: Optional[List[str]] = None):
+    toks = [t.lower() for t in re.split(r"\W+", query) if t and len(t) > 2]
+    if not toks: return []
```

Ruff (0.15.10)

[error] Multiple statements on one line (colon)

(E701)